crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -1878,7 +1911,7 @@ fn detect_similar_strings(a: &str, b: &str) -> bool {
 fn remove_server_directive_index_in_module(
     stmts: &mut Vec<ModuleItem>,
     in_action_file: &mut bool,
-    in_cache_file: &mut bool,
+    in_cache_file: &mut Option<String>,
```

Comment:
I think it might be better to use an enum here.

Reply:
Yeah that will be a follow-up.